package.json

```diff
@@ -674,6 +674,35 @@
                     }
                 }
             }
+        ],
+        "colors": [
+            {
+                "id": "problems.difficulty.badge.easy",
```

Comment:
Is it possible to refer to an existing color variable? Because if we leverage an existing color variable, the color will be changed accordingly when user switch to different themes.

Reply:
Fixed. Now, relying on inbuilt colors.